src/controllers/utils/index.ts

```diff
@@ -256,7 +256,7 @@ export const handleQueries = async ({
     id,
     version,
     searchWord,
-    keywords,
+    keywords: keywords as Keyword[],
```

Comment:
why are we casting keywords as `Keyword[]` here? shouldn't the other methods that assign a value to `keywords` automatically return the `Keyword` type?